src/mcpdotnet/Server/McpServerOptions.cs

```diff
@@ -9,36 +9,36 @@ namespace McpDotNet.Server;
 /// protocol version.
 /// <see href="https://spec.modelcontextprotocol.io/specification/2024-11-05/basic/lifecycle/">See the protocol specification for details on capability negotiation</see>
 /// </summary>
-public record McpServerOptions
+public class McpServerOptions
 {
     /// <summary>
     /// Information about this server implementation.
     /// </summary>
-    public required Implementation ServerInfo { get; init; }
+    public required Implementation ServerInfo { get; set; }
 
     /// <summary>
     /// Server capabilities to advertise to the server.
     /// </summary>
-    public ServerCapabilities? Capabilities { get; init; }
+    public ServerCapabilities? Capabilities { get; set; }
```

Comment:
If we're making all of these mutable, can you follow-up after this to make other stuff mutable as well? e.g. seems like there's no an inconsistency between McpServerOptions and McpClientOptions. We should have the same story for all these types.

Reply:
Sure. Do you want me to do that before merging or in a follow up PR?